keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -273,48 +308,85 @@ bogged down.
+_For example, an exported headless service of services name `myservice` in
+namespace `test`,  backed by pods in two clusters with cluster ids `clusterA`
+and `clusterB`, could be disambiguated by different DNS names following the
+pattern `<clusterid>.<svc>.<ns>.svc.clusterset.local`:
+`clusterA.myservice.test.svc.clusterset.local.` and
+`clusterB.myservice.test.svc.clusterset.local.`. This way the user can implement
+whatever load balancing they want (as is usually the case with headless
+services) by targeting each cluster's available backends directly._
 
 #### Diagnostics
 
-Clusters within my ClusterSet send logs/metrics to a common monitoring solution and I need to be able to identify the cluster from which a given set of events originated.
+Clusters within my ClusterSet send logs/metrics to a common monitoring solution
+and I need to be able to identify the cluster from which a given set of events
+originated.
 
 #### Multi-tenant controllers
 
-My controller interacts with multiple clusters and needs to disambiguate between them to process its business logic.
+My controller interacts with multiple clusters and needs to disambiguate between
+them to process its business logic.
 
-_For example, [CAPN's virtualcluster project](https://github.com/kubernetes-sigs/cluster-api-provider-nested) is implementing a multi-tenant scheduler that schedules tenant namespaces only in certain parent clusters, and a separate syncer running in each parent cluster controller needs to compare the name of the parent cluster to determine whether the namespace should be synced. ([ref](https://github.com/kubernetes/enhancements/issues/2149#issuecomment-768486457))._
+_For example, [CAPN's virtualcluster
+project](https://github.com/kubernetes-sigs/cluster-api-provider-nested) is
+implementing a multi-tenant scheduler that schedules tenant namespaces only in
+certain parent clusters, and a separate syncer running in each parent cluster
+controller needs to compare the name of the parent cluster to determine whether
+the namespace should be synced.
+([ref](https://github.com/kubernetes/enhancements/issues/2149#issuecomment-768486457))._
 
 
 ### `ClusterProperty` CRD
 
-The `ClusterProperty` resource provides a way to store identification related, cluster scoped information for multi-cluster tools while creating flexibility for implementations. A cluster may have multiple `ClusterProperty`s, each holding a different identification related value. Each property contains the following information:
-
-*   **Name** - a well known or custom name to identify the property.
-*   **Value** - a property-dependent string, up 128k Unicode code points (see _Note_).
+The `ClusterProperty` Kind provides a way to store identification related,
```

Comment:
We're insisting "identification related" but nothing forces that?  I could store anything here, right?